src/common/exceptions/generic-server.exception.ts

```diff
@@ -1,18 +1,16 @@
 export class GenericServerException extends Error {
   readonly name: string = 'GenericServerException';
-  readonly rawData: unknown;
   readonly message: string = 'The request could not be completed.';
 
   constructor(
     readonly status: number,
     message: string | undefined,
-    rawData: unknown,
+    readonly rawData: unknown,
```

Comment:
Changed the `GenericServerException` to always set the `rawData`, as it doesn't make sense to only set it when a `message` is present.